Fix input styles

alecperkey opened the pull request.

alecperkey committed:
remove .idea folder

alecperkey committed:
android updates for new gradle and sdk 26

alecperkey committed:
permissions check for overlay (rn debugging)

alecperkey committed:
add unstaged project setting files

alecperkey committed:
Merge pull request #1 from alecperkey/android-fcm

Android fcm

alecperkey committed:
ignore android/app/bin

alecperkey committed:
fix icons

alecperkey committed:
rebuild android with react-native init

alecperkey committed:
rn combine index entry files

alecperkey committed:
android updates for FCM

alecperkey committed:
Merge pull request #2 from alecperkey/rebuild-android-from-rn-init

Rebuild android from rn init

alecperkey committed:
ignore linebreak-style eslint

alecperkey committed:
Merge branch 'master' of https://github.com/alecperkey/chatty

alecperkey committed:
merge in app skeleton

alecperkey committed:
repalce missing icon fonts, rename 'New Group'

alecperkey committed:
fix android push notification

alecperkey committed:
merge in tweets (posts) along with Youniverse components

alecperkey committed:
add tweetLogic for returning author object

alecperkey committed:
add users (search) to api

alecperkey committed:
begin search results ui

alecperkey committed:
search results ui 2

alecperkey committed:
search results ui 3

alecperkey committed:
add navigation to screen:username-profile from component:user-list-item

alecperkey committed:
begin follower/following logic

alecperkey committed:
followed subscription system

alecperkey committed:
follow-added 2

alecperkey committed:
updateFollowed is now toggling followed

alecperkey committed:
add followed button and redesign user profile a bit

alecperkey committed:
Merge pull request #3 from alecperkey/search-results

Search results

alecperkey committed:
query both user and usernameon username-profile

alecperkey committed:
subscription args for followeds system

alecperkey committed:
add user prop to navigation, following stuff

alecperkey committed:
finalize follower toggle

alecperkey committed:
Merge pull request #4 from alecperkey/hoc-userdata

Hoc userdata

alecperkey committed:
linked user-profile-followers-screen to profile-header

alecperkey committed:
do same for followeds as following

alecperkey committed:
Merge pull request #5 from alecperkey/connections-list-item

Connections list item

alecperkey committed:
fix input height style, reapply search functionality

alecperkey committed:
interface username-search and new-conversation

alecperkey committed:
remove debuggers

alecperkey committed:
Merge pull request #6 from alecperkey/user-search-ui

User search ui

alecperkey committed:
section list implemented for new-conversation screen

alecperkey committed:
Merge pull request #7 from alecperkey/use-section-list

section list implemented for new-conversation screen

alecperkey committed:
add tiddlywiki for dev documentation

alecperkey committed:
clean up username-profile

alecperkey committed:
update aws image loading

alecperkey committed:
small change

alecperkey committed:
Merge pull request #8 from alecperkey/code-cleanup

Code cleanup

alecperkey committed:
add universe feed logic

alecperkey committed:
Merge pull request #9 from alecperkey/universe-feed

add universe feed logic

alecperkey committed:
aws

alecperkey committed:
update aws and begin youniverse feed

alecperkey committed:
add followers feed (youniverse)

alecperkey committed:
remove env from public view

alecperkey committed:
Merge pull request #10 from alecperkey/add-youniverse-feed

Add youniverse feed

alecperkey committed:
rm .env variables

alecperkey committed:
rm rootkey

alecperkey committed:
fix styles, avatar warning

srtucker22 commented:
Hey @alecperkey 

Thanks for checkout out chatty and submitting a PR! Just wanna warn outright that the organization of chatty is very weird due to its reliance on [tortilla](https://github.com/Urigo/tortilla) and makes it difficult for outside contributors to submit PRs.

Can you tell me a little bit about this PR and what it's trying to accomplish? What are some of the features you've added, bugs you've identified and fixed, etc?

Also, some of your sensitive data is showing. I would suggest (1) changing your credentials/keys on those platforms and (2) [removing commits with the sensitive data](https://help.github.com/articles/removing-sensitive-data-from-a-repository/) -- it's not enough to overwrite in a new commit

🍺

alecperkey commented:
Ahaha woops, github always defaults to point at the original (your repo)
when I push a feature branch intending to merge with my fork. Must have
missed this time. I caught some of my aws/firebase keys and git ignored/
removed/rotated them. Will double check.

Side note, I really appreciate the headstart learning your work here has
given me.

I'm working on updating fcm push notifications and Amazon file uploads this
weekend. Then swapping in postgresql (and learning other things necessary
for production).

 You can close or delete this pr

On Jan 19, 2018 9:58 AM, "Simon Tucker" <notifications@github.com> wrote:

Hey @alecperkey <https://github.com/alecperkey>

Thanks for checkout out chatty and submitting a PR! Just wanna warn
outright that the organization of chatty is very weird due to its reliance
on tortilla <https://github.com/Urigo/tortilla> and makes it difficult for
outside contributors to submit PRs.

Can you tell me a little bit about this PR and what it's trying to
accomplish? What are some of the features you've added, bugs you've
identified and fixed, etc?

Also, some of your sensitive data is showing. I would suggest (1) changing
your credentials/keys on those platforms and (2) removing commits with the
sensitive data
<https://help.github.com/articles/removing-sensitive-data-from-a-repository/>
-- it's not enough to overwrite in a new commit

🍺

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/srtucker22/chatty/pull/25#issuecomment-359042517>, or mute
the thread
<https://github.com/notifications/unsubscribe-auth/ABLv0SvAWmXlTa4vVuNmLUdJlfOezHvhks5tMNfFgaJpZM4RfILe>
.

srtucker22 commented:
No worries. Glad it was helpful!
